test/pylib/scylla_server.py

```diff
@@ -389,7 +391,7 @@ async def install_and_start(self) -> None:
                 self.cluster.append(server)
                 await server.install_and_start()
             self.keyspace_count = self._get_keyspace_count()
-        except (RuntimeError, NoHostAvailable, InvalidRequest) as e:
+        except (RuntimeError, NoHostAvailable, InvalidRequest, OperationTimedOut) as e:
```

Comment:
I don't understand the details here (will this timeout still consider the test itself to be failed), but note that your general statement that "Scylla in debug mode can easily time out on a DDL query," is indeed correct, and this is why in test/cql-pytest/conftest.py I specifically changed CQL's timeout from 10 seconds to be somewhat higher - so I'll never get these timeouts:

```python
    profile = ExecutionProfile(
       ...
        # The default timeout (in seconds) for execute() commands is 10, which
        # should have been more than enough, but in some extreme cases with a
        # very slow debug build running on a very busy machine and a very slow
        # request (e.g., a DROP KEYSPACE needing to drop multiple tables)
        # 10 seconds may not be enough, so let's increase it. See issue #7838.
        request_timeout = 120)
```

Reply:
Now I see that the next patch sets request_timeout to 300! So is this patch still needed?